database/tables/load_bond_db.sql

```diff
@@ -0,0 +1,148 @@
+  id                        bigserial   PRIMARY KEY,
+  bond_id                   varchar,
+  issuing_organization_id   varchar,
+  amount_outstanding        bigint,
+  coupon_rate               bigint,
+  first_coupon_date         date,
+  first_settlement_date     date,
+  maturity_date             date,
+  face_value                bigint,
+  coupon_type               CouponType,
+  coupon_frequency          CouponFrequency,
+  start_block_num           bigint,
+  end_block_num             bigint
+);
+
+CREATE TABLE IF NOT EXISTS corporate_dept_rates (
+  id                        bigserial   PRIMARY KEY,
+  bond_id                   varchar,
+  agency                    varchar,
+  rating                    varchar,
+  start_block_num           bigint,
+  end_block_num             bigint
+);
+
+CREATE TABLE IF NOT EXISTS organizations (
+  id                        bigserial   PRIMARY KEY,
+  organization_id           varchar,
+  industry                  varchar,
+  name                      varchar,
+  organization_type         OrganizationType,
+  pricing_source            varchar,
+  start_block_num           bigint,
+  end_block_num             bigint
+);
+
+CREATE TABLE IF NOT EXISTS authorizations (
+  id                        bigserial   PRIMARY KEY,
+  participant_public_key    varchar,
+  organization_id           varchar,
+  role                      Role,
+  start_block_num           bigint,
+  end_block_num             bigint
+);
+
+CREATE TABLE IF NOT EXISTS Receipts (
+  id                        bigserial   PRIMARY KEY,
+  payee_organization_id     varchar,
+  bond_id                   varchar,
+  payment_type              PaymentType,
+  coupon_date               date,
+  amount                    bigint,
+  timestamp                 bigint,
+  start_block_num           bigint,
+  end_block_num             bigint
+);
+
+CREATE TABLE IF NOT EXISTS holdings (
+  id                        bigserial   PRIMARY KEY,
+  owner_organization_id     varchar,
+  asset_id                  varchar,
+  asset_type                AssetType,
+  amount                    bigint,
+  start_block_num           bigint,
+  end_block_num             bigint
+);
+
+CREATE TABLE IF NOT EXISTS participants (
+  id                        bigserial   PRIMARY KEY,
+  public_key                varchar,
+  organization_id           varchar,
+  username                  varchar,
+  start_block_num           bigint,
+  end_block_num             bigint
+);
+
+CREATE TABLE IF NOT EXISTS auth (
+  public_key                varchar  PRIMARY KEY,
+  hashed_password           varchar,
+  encrypted_private_key     varchar
+);
+
+CREATE TABLE IF NOT EXISTS quotes (
+  id                        bigserial   PRIMARY KEY,
+  bond_id                   varchar,
+  organization_id           varchar,
+  ask_price                 bigint,
+  ask_quantity              bigint,
+  bid_price                 bigint,
+  bid_qty                   bigint,
```

Comment:
use `qty` or `quantity` for both bid and ask. In the docs and protos. it is `qty`

Reply:
Addressed